pageserver/src/tenant.rs

```diff
@@ -897,7 +902,7 @@ impl Tenant {
         tenant_id: TenantId,
         broker_client: storage_broker::BrokerClientChannel,
         remote_storage: Option<GenericRemoteStorage>,
-        init_done: Option<(completion::Completion, completion::Barrier)>,
+        init_order: Option<InitializationOrder>,
```

Comment:
Need to drop the repeated doc comments. The type docs explain it's purpose and usage.

Reply:
Dropped in [9fb222f](https://github.com/neondatabase/neon/pull/4399/commits/9fb222fa0917f5ecbbeffe997002db1079bb8595)